lib/membrane/video_compositor/download_compositor.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Mix.Tasks.DownloadCompositor do
+  @moduledoc """
+  Downloads VideoCompositor binaries.
+  """
+
+  use Mix.Task
+  require Membrane.Logger
+
+  @vc_version "v0.1.0-rc.2"
+
+  @impl Mix.Task
+  def run(_args) do
+    url =
+      "https://github.com/membraneframework/video_compositor/releases/download/#{@vc_version}/video_compositor_#{system_architecture()}.tar.gz"
+
+    unless File.exists?(vc_app_directory()) do
+      File.mkdir_p!(vc_app_directory())
+      Membrane.Logger.info("Downloading VideoCompositor binary")
+
+      tmp_path = "tmp"
+      File.mkdir_p!(tmp_path)
+
+      System.cmd("wget", ["-O", "tmp/video_compositor", url])
+      System.cmd("tar", ["-xvf", "tmp/video_compositor", "-C", vc_app_directory()])
+      File.rm_rf!("tmp/video_compositor")
+    end
+  end
+
+  @spec vc_app_path() :: String.t()
+  def vc_app_path() do
+    File.cwd!() |> Path.join("#{vc_app_directory()}/video_compositor/video_compositor")
+  end
+
+  defp vc_app_directory() do
+    "video_compositor_app/#{@vc_version}/#{system_architecture()}"
```

Comment:
This should probably be downloaded into `priv`, now it won't be available when deployed